toolkits/global/packages/global-forms/README.md

```diff
@@ -74,17 +69,23 @@ This component supports a wide range of standard form field attributes. For exam
 }
 ```
 
+#### Hints
+
 The `hint` property adds hint text under the main label text but _inside_ the `<label>`. This means it is automatically available to screen reader software.
 
 ```json
 "hint": "This will be the email address you used when registering"
 ```
 
-The `optional` property adds _“(optional)”_ to the label text.
+#### Required
 
-```json
-"optional": true
-```
+Fields are required by default, because you should not be asking for unnecessary data. 
+
+The `required` and `aria-required="true"` attribution is applied automatically. Unless `novalidate: true` is applied on the field’s data, browser validation will warn the user when they have not filled out the field upon submission.
+
+In the rare cases a field _is_ optional, it can be marked as such using `optional: true`. This will remove `required` and `aria-required="true"` as well as appending the text “(optional)” to the field label. This is the agreed way to communicate a field can be skipped.
```

Comment:
Just trimmed this down a bit. Suggest removing the last sentence as, *in theory* everything we do in Elements should be the agreed way to do it.

```suggestion
If you do need to ask for optional information, set `optional: true`. This will remove `required` and `aria-required="true"` as well as appending the text “(optional)” to the field label. 
```